website/docs/reference/artifacts/manifest-json.md

```diff
@@ -1,8 +1,13 @@
 ---
 title: Manifest
 ---
+<VersionBlock firstVersion="1.4">
+**Current schema**: [`v8`](https://schemas.getdbt.com/dbt/manifest/v8/index.html)
+</VersionBlock>
 
-**Current schema**: [`v6`](https://schemas.getdbt.com/dbt/manifest/v6/index.html)
+<VersionBlock lastVersion="1.3">
+**Current schema**: [`v7`](https://schemas.getdbt.com/dbt/manifest/v7/index.html)
```

Comment:
If we're going to start `VersionBlock`'ing these:
- We could also include v1.2 - v6, v1.1 - v5, v1.0 - v4
- I think the word "Current" becomes misleading, it might make more sense to start saying something like:
```suggestion
**dbt Core v1.3 produces schema**: [`v7`](https://schemas.getdbt.com/dbt/manifest/v7/index.html)
```

Reply:
@jtcohen6  Would it make more sense to do this as a list without the version blocks?